src/Routes/Routes.tsx

```diff
@@ -70,10 +70,10 @@ export const Routes = () => {
           ) : null
         }}
       </Route>
-      {appRoutes.map(route => {
+      {appRoutes.map((route, index) => {
         const MainComponent = route.main
         return (
-          <PrivateRoute path={route.path} exact key='privateRoute' hasWallet={hasWallet}>
+          <PrivateRoute key={index} path={route.path} exact hasWallet={hasWallet}>
```

Comment:
Unfortunately, the earlier commit that removed the `key` property here is causing a crash in the mobile app. Also, having each route use the same `key` may not result in expected behavior.

Another option here is to use `useMemo` on `appRoutes.map` to avoid re-rendering from iterating on each render.